native_client/deepspeech.cc

```diff
@@ -9,17 +9,35 @@
 #include "deepspeech.h"
 #include "alphabet.h"
 
+#ifndef USE_TFLITE
```

Comment:
Open to a better solution, but in the mean time, it's likely manageable this way